catalogs/catalog-jdbc-postgresql/src/test/java/com/datastrato/gravitino/catalog/postgresql/integration/test/CatalogPostgreSqlIT.java

```diff
@@ -156,16 +156,18 @@ private void createCatalog() {
     Map<String, String> catalogProperties = Maps.newHashMap();
 
     try {
-      String jdbcUrl = POSTGRESQL_CONTAINER.getJdbcUrl();
+      String jdbcUrl = POSTGRESQL_CONTAINER.getJdbcUrl(TEST_DB_NAME);
       String database = new URI(jdbcUrl.substring(jdbcUrl.lastIndexOf("/") + 1)).getPath();
```

Comment:
Are `TEST_DB_NAME` and `database` the same?

Reply:
They are the same, I'll assign `TEST_DB_NAME.toString()` to this.database directly.